_posts/2019-10-30-clojurists_together.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: Supporting Open Source Clojure Software Through Clojurists Together
+author: juholei
+excerpt: Clojurists Together is an organisation dedicated to funding and supporting Clojure open source software. Here's how it works and what our experience as a member has been like.
+tags:
+- Clojure
+- ClojureScript
+- open source
+---
+
+> Clojurists Together is an organisation run by Clojars, dedicated to funding and supporting open source software, infrastructure, and documentation that is important to the Clojure and ClojureScript community.
+
+[Clojurists Together](https://www.clojuriststogether.org) was started at the tail end of 2017 and has been running now for about two years. Solita was one of the founding company members and has been a member ever since. Clojurists Together works by collecting monthly (or yearly) donations from its individual and company members. These funds are then used to support open source software development on a quarterly basis. Each quarter a few projects are chosen for funding.
+
+The current on-going funding for Q3 of 2019 (August - November) targeted four projects: [Shadow CLJS](http://shadow-cljs.org/), [Meander](https://github.com/noprompt/meander), [Calva](https://github.com/BetterThanTomorrow/calva) and [CIDER](https://cider.mx/). Each project gets paid $9,000 over three months. [In their funding announcement posts](https://www.clojuriststogether.org/news/q3-2019-funding-announcement/) Clojurists Together provide detailed explanations of what the projects are and what their developers aim to achieve with the funding they receive. These coupled with monthly updates from the projects ([August update](https://www.clojuriststogether.org/news/august-2019-monthly-update/), [September update](https://www.clojuriststogether.org/news/september-2019-monthly-update/)) make it easy for the members and the wider Clojure community to see and understand what is being funded and how the work is progressing.
+
+In my opinion, Clojurists Together has done a great job selecting the projects to fund. For example looking at the projects currently being funded, CIDER is a massively important piece of software for clojurists. Even if you aren't using emacs with CIDER, its infrastructure in nREPL, cider-nrepl and orchard, provide a better REPL and development environment for clojurists regardless of their editor of choice. This is very well visible in another currently funded project, Calva. Calva aims to provide a first-class Clojure development environment in Visual Studio Code and builds on top of all those infrastructure pieces that are powering CIDER. As Visual Studio Code is an immensely popular editor these days, providing an easy and well-working Clojure environment for it can make Clojure much more accessible to beginners.
```

Comment:
```suggestion
In my opinion, Clojurists Together has done a great job selecting the projects to fund. For example, looking at the projects currently being funded, CIDER is a massively important piece of software for clojurists. Even if you aren't using emacs with CIDER, its infrastructure in nREPL, cider-nrepl and orchard provide a better REPL and development environment for clojurists regardless of their editor of choice. This is very well visible in another currently funded project, Calva. Calva aims to provide a first-class Clojure development environment in Visual Studio Code and builds on top of all these infrastructure pieces that are powering CIDER. As Visual Studio Code is an immensely popular editor these days, providing an easy and well-working Clojure environment for it can make Clojure much more accessible to beginners.
```

Reply:
Comma after "For example", no comma before "provider a better...", "all these infra..." instead of "all those infra..."